### PR TITLE
REST > Transactions getConsumption / assignUser schemas

### PR DESCRIPTION
**Pull request by Ninjeneer**

### CODE REVIEW

File: src/server/rest/v1/router/api/TransactionRouter.ts [hunk 1]
```diff
@@ -91,7 +107,7 @@ export default class TransactionRouter {
   }
 
   protected buildRouteTransactionConsumption(): void {
-    this.router.get(`/${ServerRoute.REST_TRANSACTIONS_CONSUMPTION}`, async (req: Request, res: Response, next: NextFunction) => {
+    this.router.get(`/${ServerRoute.REST_TRANSACTION_CONSUMPTION}`, async (req: Request, res: Response, next: NextFunction) => {
```
Comment (LucasBrazi06) [inline src/server/rest/v1/router/api/TransactionRouter.ts:110]: Consumption is plural.

File: src/assets/server/rest/v1/docs/e-mobility-oas.json [hunk 2]
```diff
@@ -956,6 +1512,43 @@
         }
       }
     },
+    "/api/transactions/status/unassigned/count": {
```
Comment (LucasBrazi06) [inline src/assets/server/rest/v1/docs/e-mobility-oas.json:1515]: Weird this one :S
Where is it used?

File: src/assets/server/rest/v1/schemas/transaction/transaction-get-consumption.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (LucasBrazi06) [inline src/assets/server/rest/v1/schemas/transaction/transaction-get-consumption.json:1]: Consumption in plural

File: src/server/rest/v1/validator/TransactionValidator.ts [hunk 4]
```diff
@@ -51,4 +55,16 @@ export default class TransactionValidator extends SchemaValidator {
     this.validate(this.transactionPushCDR, data);
     return data;
   }
+
+  public validateTransactionGetConsumptionReq(data: any): HttpConsumptionFromTransactionRequest {
```
Comment (LucasBrazi06) [inline src/server/rest/v1/validator/TransactionValidator.ts:59]: Consumption in plural

File: src/types/Server.ts [hunk 5]
```diff
@@ -493,18 +493,23 @@ export enum ServerRoute {
...
+  REST_TRANSACTIONS_ACTIVE = 'transactions/status/active',
+  REST_TRANSACTIONS_COMPLETED = 'transactions/status/completed',
+  REST_TRANSACTIONS_UNASSIGNED_COUNT = 'transactions/status/unassigned/count',
   REST_TRANSACTION = 'transactions/:id',
   REST_TRANSACTIONS_EXPORT = 'transactions/action/export',
   REST_TRANSACTION_CDR = 'transactions/:id/ocpi/cdr',
-  REST_TRANSACTION_CDR_EXPORT = 'transactions/:id/ocpi/cdr/action/export',
-  REST_TRANSACTIONS_CONSUMPTION = 'transactions/:id/consumptions',
+  REST_TRANSACTION_CDR_EXPORT = 'transactions/:id/ocpi/cdr/export',
+  REST_TRANSACTION_CONSUMPTION = 'transactions/:id/consumptions',
```
Comment (LucasBrazi06) [inline src/types/Server.ts:504]: It's in plural in the path but not in the constant ;-)

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]

File: src/assets/server/rest/v1/docs/e-mobility-oas.json (hunk 2 above)
Comment (Ninjeneer) [reply src/assets/server/rest/v1/docs/e-mobility-oas.json:1515]: It is the "UNASSIGNED_TRANSACTIONS_COUNT" action, calling the "handleGetUnassignedTransactionsCount" method in TransactionService
After checking, it seems not to be used neither in dashboard or mobile app :)

Comment (LucasBrazi06) [reply src/assets/server/rest/v1/docs/e-mobility-oas.json:1515]: Okay, you can remove it then :)

Comment (Ninjeneer) [reply src/assets/server/rest/v1/docs/e-mobility-oas.json:1515]: My bad, it used in the Tag components of the Dashboard :)

Comment (LucasBrazi06) [reply src/assets/server/rest/v1/docs/e-mobility-oas.json:1515]: Okay, we keep it then.

**Review by LucasBrazi06 — APPROVED** [review]
GUT !

**Pull request merged**